docs/pages/build-reference/apk.mdx

```diff
@@ -8,16 +8,17 @@ The default file format used when building Android apps with EAS Build is an [An
 
 ### Managed projects
 
-By default, EAS Build produces Android App Bundle, you can change it by modifying the [eas.json](/build/eas-json.mdx) file:
+By default, EAS Build produces Android App Bundle, you can change it by modifying the [eas.json](/build/eas-json.mdx) file by either:
 
-- setting `buildType` to `apk`
-- setting `developmentClient` to true
+- setting `buildType` to `apk` or
+- setting `developmentClient` to `true` or
 - setting `gradleCommand` to `:app:assembleRelease`, `:app:assembleDebug` or any other gradle command that produces APK
 
+For example:
 ```json
 {
   "build": {
-    "preview": {
+    "preview1": {
```

Comment:
```suggestion
    "preview": {
```

For clarity, since the default **eas.json** already generates a `"preview"` profile, I think we should keep this to `"preview"`.